spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/ReleaseService.java

```diff
@@ -185,9 +188,18 @@ public Release install(Release release) {
 		Map<String, Object> mergedMap = ConfigValueUtils.mergeConfigValues(release.getPkg(), release.getConfigValues());
 		// Render yaml resources
 		String manifestData = ManifestUtils.createManifest(release.getPkg(), mergedMap);
+		DumperOptions dumperOptions = new DumperOptions();
+		dumperOptions.setDefaultScalarStyle(DumperOptions.ScalarStyle.DOUBLE_QUOTED);
+		dumperOptions.setPrettyFlow(true);
```

Comment:
Don't think that pretty print is necessary here.

Reply:
yea, saves a bit of space as it gets stored in the DB, but maybe helps with debugging?